README.md

```diff
@@ -529,6 +530,34 @@ export function MyComponent() {
 - `href`: `<string>` href string
 - `lang`: `<string>` language
 
+### documentLang
+
+📦**Size**: ???
+
+Helper to automatically update the document language on routing.
```

Comment:
This is not entirely true. Document code does not run on the client-side. So the only thing the helper can do is load the language in the lang attribute of the HTML tag.

The logic to change this attribute on routing is inside `next/Router` and `next/Link`,

Reply:
I see, what do you think about this text: "Helper to automatically retrieve the locale for the current route."?

Reply:
Or should the text explicitly mention the "_document.js" file or the HTML tag and lang attribute?

Reply:
Ahh sure we need to add "_document.js", because it uses `__NEXT_DATA__`, so what about this: `Helper to retrieve the locale from the current route and keep the document language in sync on routing.`

Reply:
Maybe just keep it simple it's enough: `It's a helper to get the page language inside _document.js` 😊